apps/src/applab/locale-do-not-import.js

```diff
@@ -8,4 +8,9 @@
  */
 // locale for applab
 
-export default window.locales.applab_locale;
```

Comment:
I wonder why this file doesn't use `safeLoadLocale`...

Reply:
probably an oversight on my part when I originally implemented that helper. I certainly don't recall intentionally excluding applab